src/pages/Exchange/MainArea/Orders/index.tsx

```diff
@@ -100,11 +103,21 @@ const Orders: SFC = ({className}) => {
 
       const primaryCurrencyTicker = getCurrencyTicker(primary_currency);
       const secondaryCurrencyTicker = getCurrencyTicker(secondary_currency);
+      const [date, time] = longDate(created_date).split('at');
```

Comment:
Just thinking, would this break if we ever replace `at` with something else in future in `longDate` function?
or I'm being too paranoid? 🤣 

I think it would not break, the UI will just slightly change